server-common/src/test/java/org/apache/kafka/timeline/TimelineHashSetTest.java

```diff
@@ -60,15 +57,13 @@ public void testIteration() {
         set.add("b");
         set.add("c");
         set.add("d");
-        assertTrue(set.retainAll(Arrays.asList("a", "b", "c")));
-        assertFalse(set.retainAll(Arrays.asList("a", "b", "c")));
-        assertFalse(set.removeAll(Arrays.asList("d")));
+        assertTrue(set.retainAll(asList("a", "b", "c")));
```

Comment:
Are these Arrays.asList -> asList changes needed? I think in most places we import Arrays rather than the static method import